packages/react-components/components/input/src/textarea.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable react/forbid-foreign-prop-types */
+
+import { Ref, TextArea as SemanticTextArea } from "semantic-ui-react";
+import { bool, func, number, object, oneOfType } from "prop-types";
+import { forwardRef, useEffect, useImperativeHandle, useRef } from "react";
+import { isNil } from "lodash";
+import { mergeClasses, throwWhenUnsupportedPropIsProvided } from "@orbit-ui/react-components-shared";
+
+const UNSUPPORTED_PROPS = [];
+
+const propTypes = {
+    /**
+     * Whether or not the textarea should autofocus on render.
+     */
+    autofocus: bool,
+    /**
+     * Delay before trying to autofocus.
+     */
+    autofocusDelay: number,
+    /**
+     * Whether or not the textarea is invalid.
+     */
+    error: bool,
+    /**
+     * Whether or not the textarea is valid.
+     */
+    success: bool,
+    /**
+     * Whether or not a user is currently interacting with the textarea.
+     */
+    focused: bool,
+    /**
+     * Whether or not the textarea is transparent.
+     */
+    transparent: bool,
+    /**
+     * @ignore
+     */
+    forwardedRef: oneOfType([object, func])
+};
+
+const defaultProps = {
+    autofocus: false,
+    error: false,
+    success: false,
+    focused: false,
+    transparent: false
+};
+
+function getTextAreaElement(textAreaRef) {
+    return textAreaRef.current;
+}
+
+function focus(textAreaRef) {
+    if (!isNil(textAreaRef.current)) {
+        getTextAreaElement(textAreaRef).focus();
+    }
+}
+
+function useDelayedAutofocus(autofocus, autofocusDelay, disabled, textAreaRef) {
+    useEffect(() => {
+        let timeoutId;
+
+        if (autofocus && !disabled && !isNil(autofocusDelay)) {
+            timeoutId = setTimeout(() => {
+                focus(textAreaRef);
+            }, autofocusDelay);
+        }
+
+        return () => {
+            if (!isNil(timeoutId)) {
+                clearTimeout(timeoutId);
+            }
+        };
+    }, [autofocus, autofocusDelay, disabled, textAreaRef]);
+}
+
+export function PureTextArea(props) {
+    const { autofocus, autofocusDelay, className, disabled, error, success, focused, transparent, children, forwardedRef, ...rest } = props;
+
+    throwWhenUnsupportedPropIsProvided(props, UNSUPPORTED_PROPS, "@orbit-ui/react-input");
+
+    const textAreaRef = useRef();
+
+    useImperativeHandle(forwardedRef, () => getTextAreaElement(textAreaRef));
```

Comment:
you don't have to use useImperativeHandle in this case, simply pass forwardRef to `Ref` instead of creating a proxy ref.

```
<Ref innerRef={forwardRef} ...
```